src/Map/composer.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "symfony/ux-map",
+    "type": "symfony-bundle",
+    "description": "Easily embed interactive maps in your Symfony application",
+    "keywords": [
+        "symfony-ux",
+        "map",
+        "markers",
+        "maps"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Hugo Alliaume",
+            "email": "hugo@alliau.me"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Symfony\\UX\\Map\\": "src/"
+        },
+        "exclude-from-classmap": []
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Symfony\\UX\\Map\\Tests\\": "tests/"
+        }
+    },
+    "require": {
+        "php": ">=8.3",
+        "symfony/stimulus-bundle": "^2.18.12"
```

Comment:
Latest version is [2.18.1](https://github.com/symfony/stimulus-bundle/releases/tag/v2.18.1) ?

Reply:
Thanks!